chargeamps/__init__.py

```diff
@@ -1,3 +1,3 @@
-import pkg_resources
+from importlib_metadata import version
```

Comment:
Is this not supposed to be `importlib.metadata` ?

Reply:
You actually have both options available to you. I'm not sure about the benefits/drawbacks for each in the long run. I'll update to `importlib.metadata` and let you decide the way forward.